### ISSUE TITLE
How to pass organization header?

### ISSUE THREAD

**Issue by Boscop** [dongri/openai-api-rs#25]
Thanks for making this crate, it seems very useful :)

Btw, on https://platform.openai.com/account/api-keys it says:

![image](https://github.com/dongri/openai-api-rs/assets/535593/ba984bca-0c04-4f8c-9536-27735c38b023)

https://platform.openai.com/docs/api-reference/requesting-organization says:

> ### [Requesting organization](https://platform.openai.com/docs/api-reference/requesting-organization)
> For users who belong to multiple organizations, you can pass a header to specify which organization is used for an API request. Usage from these API requests will count against the specified organization's subscription quota.
Example curl command:
```sh
curl https://api.openai.com/v1/models \
  -H "Authorization: Bearer $OPENAI_API_KEY" \
  -H "OpenAI-Organization: org-Q4qnf6HKZtLiqUySAB41oMeY"
```

Could you please add this header functionality to this crate, so that we can select the org? :)

**Comment by dongri** [comment]
https://github.com/dongri/openai-api-rs/pull/28
Add new_with_organization

@Boscop The `new_with_organization` function is available in v0.1.15

**Comment by Boscop** [comment]
Thanks :)
